pkgs/development/tools/language-servers/nls/default.nix

```diff
@@ -9,13 +9,20 @@ rustPlatform.buildRustPackage {
 
   inherit (nickel) src version nativeBuildInputs;
 
-  cargoHash = "sha256-UGfc5cr6vl10aCVihOEEZktF8MzT56C9/wSvSQhCiVs=";
+  cargoLock = {
+    lockFile = ./Cargo.lock;
```

Comment:
These cargo lock files are duplicated. Should I keep them self-standing or reference nickel's `Cargo.lock` file instead?

Tbh `nls` could also be an additional output of `nickel` and then aliased to `pkgs.nls`. It would save some compilation time for sure.

Reply:
> These cargo lock files are duplicated. Should I keep them self-standing or reference the Cargo.lock file from the nickel package instead?

As they are bitwise identical, git will store them as the very same blob, so there's no need (IMO) deduplicating them.